src/Models/ApplicantModel.php

```diff
@@ -70,11 +70,13 @@ public function storeApplicant(array $applicant)
         if ($result) {
             $id = $this->db->lastInsertId();
             foreach ($applicant['cohort'] as $cohortId) {
-                $query2 = $this->db->prepare('INSERT INTO `course_choice` (`coursesid`, `applicantsid`) VALUES (?,?)');
-                $query2->execute([$cohortId, $id]);
+                $courseChoiceQuery = $this->db->prepare(
+                    'INSERT INTO `course_choice` (`courseId`, `applicantId`) VALUES (?,?)'
+                );
+                $courseChoiceQuery->execute([$cohortId, $id]);
             }
-            $query2 = $this->db->prepare('INSERT INTO `applicants_additional` (`id`) VALUES (?)');
-            return $query2->execute([$id]);
+            $courseChoiceQuery = $this->db->prepare('INSERT INTO `applicants_additional` (`id`) VALUES (?)');
```

Comment:
wrong variable name, this query is different to the above one. can we have a better variable name please